src/test/java/de/tum/in/www1/artemis/bonus/BonusFactory.java

```diff
@@ -10,13 +10,13 @@
 public class BonusFactory {
 
     /**
-     * Generates a Bonus instance with given arguments.
+     * Generates a Bonus with the given arguments.
      *
-     * @param bonusStrategy         of bonus
-     * @param weight                of bonus
-     * @param sourceGradingScaleId  of sourceGradingScale of bonus
-     * @param bonusToGradingScaleId of bonusToGradingScale bonus
-     * @return a new Bonus instance associated with the grading scales corresponding to ids bonusToGradingScaleId and bonusToGradingScaleId.
+     * @param bonusStrategy         The bonus strategy of the bonus
+     * @param weight                The weight of the bonus
+     * @param sourceGradingScaleId  The id of the grading scale that is used as the source for the bonus
```

Comment:
Sounds a bit complex. Maybe: "The id of the source grading scale for the Bonus" ?

Reply:
implemented